source/documentation/modules/math.html.md.erb

```diff
@@ -0,0 +1,494 @@
+  [using `...`][].
+
+  [using `...`]: ../at-rules/function#passing-arbitrary-arguments
+
+  <% example(autogen_css: false) do %>
+    @debug math.max(1px, 4px); // 4px
+
+    $widths: 50px, 30px, 100px;
+    @debug math.max($widths...); // 100px
+    ===
+    @debug math.max(1px, 4px)  // 4px
+
+    $widths: 50px, 30px, 100px
+    @debug math.max($widths...)  // 100px
+  <% end %>
+<% end %>
+
+
+<% function 'math.min($number...)', 'min($number...)', returns: 'number' do %>
+  Returns the lowest of one or more numbers. A list of numbers can be passed
+  [using `...`][].
+
+  [using `...`]: ../at-rules/function#passing-arbitrary-arguments
+
+  <% example(autogen_css: false) do %>
+    @debug math.min(1px, 4px); // 1px
+
+    $widths: 50px, 30px, 100px;
+    @debug math.min($widths...); // 30px
+    ===
+    @debug math.min(1px, 4px)  // 1px
+
+    $widths: 50px, 30px, 100px
+    @debug math.min($widths...)  // 30px
+  <% end %>
+<% end %>
+
+
+<% function 'math.round($number)',
+            'round($number)',
+            returns: 'number' do %>
+  Rounds `$number` to the nearest whole number.
+
+  <% example(autogen_css: false) do %>
+    @debug math.round(4); // 4
+    @debug math.round(4.2); // 4
+    @debug math.round(4.9); // 5
+    ===
+    @debug math.round(4)  // 4
+    @debug math.round(4.2)  // 4
+    @debug math.round(4.9)  // 5
+  <% end %>
+<% end %>
+
+
+## Distance Functions
+
+<% function 'math.abs($number)',
+            'abs($number)',
+            returns: 'number' do %>
+  Returns the [absolute value][] of `$number`. If `$number` is negative, this
+  returns `-$number`, and if `$number` is positive, it returns `$number` as-is.
+
+  [absolute value]: https://en.wikipedia.org/wiki/Absolute_value
+
+  <% example(autogen_css: false) do %>
+    @debug math.abs(10px); // 10px
+    @debug math.abs(-10px); // 10px
+    ===
+    @debug math.abs(10px) // 10px
+    @debug math.abs(-10px) // 10px
+  <% end %>
+<% end %>
+
+
+<% function 'math.hypot($number...)', returns: 'number' do %>
+  Returns the length of the *n*-dimensional vector that has components equal to
+  each `$number`.
+
+  The numbers must either all have compatible units, or all be unitless. And
+  since the numbers' units may differ, the output takes the unit of the first
+  number.
+
+  A list of numbers can be passed [using `...`][].
+
+  [using `...`]: ../at-rules/function#passing-arbitrary-arguments
+
+  <% example(autogen_css: false) do %>
+    @debug math.hypot(3, 4); // 5
+
+    $lengths: 1in, 10cm, 50px;
+    @debug math.hypot($lengths...); // 4.0952775683in
+    ===
+    @debug math.hypot(3, 4) // 5
+
+    $lengths: 1in, 10cm, 50px
+    @debug math.hypot($lengths...) // 4.0952775683in
+  <% end %>
+<% end %>
+
+
+## Exponential Functions
+
+<% function 'math.log($number, $base: null)',
+            returns: 'number' do %>
+  Returns the logarithm of `$number` with respect to `$base`. If `$base` is
+  [`null`][], the [natural log][] is calculated.
```

Comment:
I think linking `null` here is overkill; we don't generally link it elsewhere.

Reply:
Done.